src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> list = new ArrayList<>();
+        list.add(new Bulldozer(220, "ROBO-20"));
+        list.add(new Bulldozer(200, "ROBO-20PRO"));
+        return list;
```

Comment:
Also you can use List.of for creating immutable list

Reply:
done